### PR TITLE
Adding pytest_parallel_mpi package

### PR DESCRIPTION
**Pull request by MicK7**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by MicK7** [comment]
> Feedstock with the name pytest-parallel exists in conda-forge. Is it the same as this package (pytest_parallel)?

No it is a different package with a different backend. pytest-parallel use multithreading while pytest_parallel use distributed MPI

**Review by xhochy — CHANGES_REQUESTED** [review]
Please choose a different name. They are too equal to each other to differentiate.

**Comment by MicK7** [comment]
> Please choose a different name. They are too equal to each other to differentiate.

@xhochy  I changed the package name to "pytest_parallel_mpi" to be more expressive

**Comment by MicK7** [comment]
@conda-forge/help-python please review

**Comment by MicK7** [comment]
https://github.com/orgs/conda-forge/teams/help-python can this topic be merged ?

**Comment by MicK7** [comment]
https://github.com/orgs/conda-forge/teams/help-python up for  a review

**Comment by MicK7** [comment]
@xhochy can you approve the changes ?

File: recipes/pytest_parallel_mpi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - pytest ==6.2.5
```
Comment (xhochy) [inline recipes/pytest_parallel_mpi/meta.yaml:23]: Where is this strict requirement come from? It doesn't look like upstream has it.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/pytest_parallel_mpi/meta.yaml (hunk 1 above)
Comment (MicK7) [reply recipes/pytest_parallel_mpi/meta.yaml:23]: Upstream has this strict requirement for the release but not the current master branch (development not released.)

**Review by xhochy — APPROVED** [review]

**Comment by xhochy** [comment]
You should also upload this to PyPI to reserve the name.

**Pull request merged**